.npmignore

```diff
@@ -1,4 +1,3 @@
-types/ethers-v5
```

Comment:
the increase in size could also be from including the types? Im not sure we need to include these?

Reply:
If they are not included, the source map doesnt work